base/linalg/symmetric.jl

```diff
@@ -332,9 +332,10 @@ end
 
 factorize(A::HermOrSym) = bkfact(A)
 
-det(A::RealHermSymComplexHerm) = real(det(bkfact(A)))
-det(A::Symmetric{<:Real}) = det(bkfact(A))
-det(A::Symmetric) = det(bkfact(A))
```

Comment:
Ambiguities. Actually, the LU-fallback is not necessary since we end up [here](https://github.com/JuliaLang/julia/blob/d9903b1f04c29ac0f434f959661a5d2d40ae0b35/base/linalg/generic.jl#L1224-L1230) without it, so might as well remove it completely.